### ISSUE TITLE
No way to get count of points that shown after setExtreme?

### ISSUE THREAD

**Issue by PAPIONbit** [highcharts/highcharts#12017]
### Expected behavior:
After setting the extreme, in **afterSetExtremes** i just can get count and detail of viewed data from **event**
+ there be a function for it.

### Actual behavior:
I should count all of my data with a loop and use **isInside**!

### Product version:
Highstock 7.2.0

### Explained:
[https://stackoverflow.com/q/57981487/7514010](https://stackoverflow.com/q/57981487/7514010)

Best Regards

**Comment by pawelfus** [comment]
Hi @PAPIONbit 

You can check `series.processedXData.length` arrays. That array has number of **rendered** points (not inside the current view). For example when zooming in line series, there will be two extra points (one point before `min` and one after `max`).

Note that is only true when [cropThreshold](https://api.highcharts.com/highcharts/series.line.cropThreshold) is exceeded and when series **requires sorted data**. For example `scatter` or `bubble` series don't require sorted data, so there's no information which points are within the extremes.

Of course, this only applies to xAxis extremes. Setting extremes on yAxis (or zooming) does not change number of available points and you need to process data anyway.

To sum up, we don't need anywhere number of visible points, so we don't count those and you need to use `for` loop to count them.

**Comment by PAPIONbit** [comment]
Yes, exactly.
The code should be something like this :
```
    chart: {
        events: {
            render: function() {
                console.log(this.series[0].points.length)
            }
        }
    },
```
and the **points** object have enough data for analyse.

actually other developers in stackoverflow, like :
[https://stackoverflow.com/questions/25936630/highcharts-how-to-get-the-data-points-in-the-zoomed-window](https://stackoverflow.com/questions/25936630/highcharts-how-to-get-the-data-points-in-the-zoomed-window)
or
[https://stackoverflow.com/questions/29578322/highchart-get-total-of-visible-series-data-after-setextremes](https://stackoverflow.com/questions/29578322/highchart-get-total-of-visible-series-data-after-setextremes)
and not enough example in references of Highcharts.com:
[https://api.highcharts.com/highstock/chart.events.redraw](https://api.highcharts.com/highstock/chart.events.redraw)
made me confused.
(+) Thanks for the noticing about **cropThreshold**

Best Regards
